diffraction/powder_reduction.py

```diff
@@ -339,41 +376,49 @@ def process_vanadium_data(vanadium, empty_instr, lambda_binning, calibration=Non
                                            lambda_binning,
                                            **absorp_vana)
 
-        # the 3 following lines of code are to place info about source and sample
-        # position at the right place in the correction dataArray in order to
-        # proceed to the normalization
+        # the 3 following lines of code are to place info about source and
+        # sample position at the right place in the correction dataArray in 
+        # order to proceed to the normalization
 
-        del correction.coords['source-position']
-        del correction.coords['sample-position']
+        del correction.coords['source_position']
+        del correction.coords['sample_position']
         del correction.coords['position']
 
-        correction = sc.rebin(correction,
-                              'wavelength',
-                              sc.Variable(['wavelength'],
-                                          values=vana_red.coords['wavelength'].values,
-                                          unit=sc.units.angstrom))
+        lambda_min, lambda_max, number_bins = lambda_binning
 
-        vana_red /= correction
+        edges_lambda = sc.Variable(['wavelength'],
+                                    unit=sc.units.angstrom,
+                                    values=np.linspace(lambda_min,
```

Comment:
Note that in the next release of scipp, we have included numpy-like helpers `sc.linspace` etc (see https://github.com/scipp/scipp/pull/1871)